docs/en/guides/sre/s3-multi-region.md

```diff
@@ -0,0 +1,183 @@
+<clickhouse>
+   <merge_tree>
+        <allow_remote_fs_zero_copy_replication>false</allow_remote_fs_zero_copy_replication>
+   </merge_tree>
+</clickhouse>
+```
+
+## Define a cluster
+
+ClickHouse cluster(s) are defined in the `<remote_servers>` section of the configuration.  In this sample one cluster, `cluster_1S_2R`, is defined and it consists of a single shard with two replicas.  The replicas are located on the hosts `chnode1` and `chnode2`.
+
+```xml title="/etc/clickhouse-server/config.d/remote-servers.xml"
+<clickhouse>
+    <remote_servers replace="true">
+        <cluster_1S_2R>
+            <shard>
+                <replica>
+                    <host>chnode1</host>
+                    <port>9000</port>
+                </replica>
+                <replica>
+                    <host>chnode2</host>
+                    <port>9000</port>
+                </replica>
+            </shard>
+        </cluster_1S_2R>
+    </remote_servers>
+</clickhouse>
+```
+
+When working with clusters it is handy to define macros that populate DDL queries with the cluster, shard, ??? settings.  This sample allows you to write ..... (give example DDL using `{cluster}` etc.)
+
+```xml title="/etc/clickhouse-server/config.d/macros.xml"
+<clickhouse>
+    <distributed_ddl>
+            <path>/clickhouse/task_queue/ddl</path>
+    </distributed_ddl>
+    <macros>
+        <cluster>cluster_1S_2R</cluster>
+        <shard>1</shard>
+        <replica>replica_1</replica>
+    </macros>
+</clickhouse>
+```
+
+ClickHouse Keeper is responsible for coordinating the replication of data across the ClickHouse nodes.  To inform ClickHouse about the ClickHouse Keeper nodes add a configuration file to each of the ClickHouse nodes.
+
+```xml title="/etc/clickhouse-server/config.d/use_keeper.xml"
+<clickhouse>
+    <zookeeper>
+        <node index="1">
+            <host>chnode1</host>
+            <port>9181</port>
+        </node>
+        <node index="2">
+            <host>chnode2</host>
+            <port>9181</port>
+        </node>
+        <node index="3">
+            <host>keepernode1</host>
+            <port>9181</port>
+        </node>
+    </zookeeper>
+</clickhouse>
+```
+
+## Configure networking
+
+All three servers must listen for network connections so that they can communicate between the servers and with S3.  This is configured in `/etc/clickhouse-server/config.d/`.  Here is a sample that configures ClickHouse and ClickHouse Keeper to listen on all IP v4 interfaces.  see the documentation or the default configuration file `/etc/clickhouse/config.xml` for more information.
+
+```xml title="/etc/clickhouse-server/config.d/networking.xml"
+<clickhouse>
+    <listen_host>0.0.0.0</listen_host>
+</clickhouse>
+```
+
+## Testing
+
+To do
+
+- Create a table that is replicated and uses the S3 storage
+- Add data
+- Verify that data is stored in S3
+- Shutdown one S3 bucket?
+- verify data avaialability
```

Comment:
This tutorial does not cover opening ports in AWS (for data replication, for communication with Keepers, for communication between Keepers, for connecting to ClickHouse from the clients).

Reply:
Good point, I have a list of ports used from support services and will add that to the docs and refer to it.